app/assets/stylesheets/scaffolds.scss

```diff
@@ -1,12 +1,14 @@
+@import 'vars/variables';
+
 @import 'variables';
 @import 'mixins';
 @import 'fonts';
 
 body {
   @include themeable(background, theme-background, $lightest-gray);
   @include themeable(color, theme-color, $black);
-  font-family: $helvetica;
-  font-size: 18px;
+  font-family: $ff-default;
+  font-size: 16px; // 1rem
```

Comment:
I bring back "18px" in [other place](https://github.com/thepracticaldev/dev.to/pull/6334/files#diff-0798fbc2c93da3f42838f733d383f800R66), just for now to not brake things visually.

Reply:
Is setting this to 16px necessary? It's the default value in browsers.

Reply:
I think default browsers' font size is actually 16px, not 18px: https://stackoverflow.com/a/29512572/1147887

The reason for switching it from 18 to 16 is  because we're using lots of rems here and there which are obviously based on that size. So I'd rather "fix" it sooner than later.